MSAL/src/MSALPublicClientApplication+Internal.h

```diff
@@ -27,8 +27,8 @@
 
 #import "MSALPublicClientApplication.h"
 
-// TODO: Remove these defaults once uid/client info is in prod
-#define DEFAULT_SLICE_PARAMS @"slice" : @"testslice", @"uid" : @"true"
+// If modifying this make sure UT_SLICE_PARAMS_QUERY matches
```

Comment:
I'm hesitant to remove it, because the code was added in the first place because sometimes it's easier to just hard code MSAL to a specific test slice then modify the code using MSAL. It wouldn't surprise me if it sees some utility in our testing.